src/Azure.Functions.Cli/Helpers/WorkerRuntimeLanguageHelper.cs

```diff
@@ -55,7 +55,7 @@ public static class WorkerRuntimeLanguageHelper
             { Constants.Languages.TypeScript, new [] { "ts" } },
             { Constants.Languages.Python, new [] { "py" } },
             { Constants.Languages.Powershell, new [] { "pwsh" } },
-            { Constants.Languages.CSharp, new [] { "csharp", "dotnet", "dotnet-isolated", "dotnetIsolated" } },
+            { Constants.Languages.CSharpIsolated, new [] { "csharp", "dotnet", "dotnet-isolated", "dotnetIsolated" } },
```

Comment:
I believe there should be separate entries here: one with `{ Constants.Languages.CSharp, new [] { "csharp", "dotnet" } }` and another with `{ Constants.Languages.CSharpIsolated, new [] { "dotnet-isolated", "dotnetIsolated" } }`

Reply:
Yes that's probably true, otherwise it won't detect in-proc properly

Reply:
Done but I've had to move `csharp` out of the CSharpIsolated constant, otherwise it'll explode later on in `WorkerRuntimeStringToLanguage`, but I'm not quite sure how they all hang together to know if that's a problem (hasn't seem to be in my testing)